lib/al/Library/String/StringUtil.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+namespace al {
+template <typename T>
+bool isEqualString(const T, const T);
```

Comment:
No, that's too broad - not a templated function, but actually three separate functions (`char`, `char16_t` and `SafeString`)